hepcrawl/spiders/aps_spider.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, from_date=None, until_date=None,
         self.until_date = until_date
         self.per_page = per_page
         self.date = date
+        self.aps_token = aps_token
```

Comment:
maybe store the authorization header instead, as that's what you're using everywhere?
```suggestion
        self.auth_header = "Bearer {}".format(aps_token)
```